about.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container-md d-flex flex-column align-items-center my-5">
+      <h1
+        class="about-header display-2 fw-bolder text-uppercase text-center my-2"
+      >
+        "Jah Malet Fetari" International Chess Tournament 2023
+      </h1>
+      <div class="sub-heading text-center my-2">
+        <p class="fs-5">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat
+          dignissimos dolore recusandae consectetur iste, sapiente eaque harum
+          impedit dolores corrupti ab minus quis, molestiae excepturi sit cumque
+          tempora iure. Explicabo impedit dolores magnam dicta unde ea eius
+          error! Dolorem similique fuga molestiae eveniet atque praesentium
+          doloribus libero nobis totam tempore.
+        </p>
+      </div>
+      <div class="w-50 my-4">
+        <p class="fs-4 text-center">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Explicabo,
+          doloribus!
+        </p>
+      </div>
+      <div class="about-section-2 d-flex flex-column align-items-center">
+        <h3 class="text-center">
+          "Jah Malet Fetari" International Chess Tournament 2023 Logo
+        </h3>
+        <hr class="red-hr" />
+        <p class="fs-5 text-center">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quas natus
+          beatae voluptate perspiciatis molestiae perferendis!
+        </p>
+        <img src="assets/mozilla.svg" class="about-logo px-5 py-3" />
+      </div>
+      <hr class="about-section-hr my-5" />
```

Comment:
- [ ] Please 😃 try to stick to the Behance at the color of the text, so please try to make this text to be like the Behance and add the correct color 💯


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/225009695-fc744ee0-a723-4012-b665-6fc82ff9ec93.png) |   ![image](https://user-images.githubusercontent.com/83360636/225009563-484f487b-a810-49cd-b8d4-312bc31eb6f9.png)   |